apps/web/public/locales/es.json

```diff
@@ -989,22 +990,22 @@
   "URLs must have between 2 and 40 characters, and can only have lower case letters, numbers, and dashes.": "Las URL deben tener entre 2 y 40 caracteres y solo pueden tener letras minúsculas, números y guiones.",
   "Unavailable": "No disponible",
   "Unblock": "Desbloquear",
-  "Unlink": "Desvincular",
   "Unit term": "Unidad",
+  "Unlink": "Desvincular",
   "Unit term plural": "Unidades",
   "Unpin": "Desprender",
-  "Unread": "No leído",
   "Unpublish this track": "No publicar esta ruta",
   "Unpublished": "No publicado",
+  "Unread": "No leído",
```

Comment:
you still need to show me how to get the automated updates to these files to work, I couldn't figure it out

Reply:
yeah, its whatever I did all that time ago to setup i18n-ally, the vscode extension... I do need to review that